test/bigtest/helpers/Harness.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { IntlProvider } from 'react-intl';
+
+import translations from '../../../translations/ui-agreements/en';
+
+// mimics the StripesTranslationPlugin in @folio/stripes-core
+function prefixKeys(obj) {
+  const res = {};
+  for (const key of Object.keys(obj)) {
+    res[`ui-agreements.${key}`] = obj[key];
+  }
+  return res;
+}
+
+class Harness extends React.Component {
```

Comment:
It looks like the point of this is just for providing translations. [Would this PR help us avoid adding custom test runner code to our repo?](https://github.com/folio-org/stripes-components/pull/1090)